Create ui-c8y-1023-2-4-quick-links-widget-shows-strange-text-keep-original-for-open-api.md (#3934)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-2-4-quick-links-widget-shows-strange-text-keep-original-for-open-api.md

---
date: ""

> title: Quick Links widget shows incorrect text for an OpenAPI link

Reviewer: ```suggestion
title: Quick Links widget displays correct text for link to OpenAPI specification
```

> title: Quick Links widget shows strange text Keep Original for OpenAPI (#10002) [GRAFT][release/cd] (#10336)

Reviewer 2: ```suggestion
title: Quick Links widget shows incorrect text for an OpenAPI link
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64728
version: 1023.2.4
---

> Co-authored-by: Igor Reznichenko <igor.reznichenko@cumulocity.com>

Reviewer 2: ```suggestion
In the Cockpit and Device Management applications, the Quick Links widget is used to provide easy access to useful links. However, the widget incorrectly displayed "Keep Original" instead of the proper OpenAPI link name. This issue has now been resolved. The Quick Links widget will display the correct link name for OpenAPI specifications.
```

> In the Cockpit and Device Management application, the "Quick links" widget, which provides easy access to useful links, did not display the link name for the OpenAPI specification properly. This issue has been resolved. The "Quick links" widget now displays the correct link name "OpenAPI specification".

Reviewer: ```suggestion
In the Cockpit and Device Management application, the "Quick links" widget, which provides easy access to useful links, did not display the link text for the OpenAPI specification properly. This issue has been resolved. The "Quick links" widget now displays the correct link text "OpenAPI specification".
```

> In the Cockpit and Device Management applications, the Quick Links widget is used to provide easy access to useful links. However, the widget incorrectly displayed "Keep Original" instead of the proper OpenAPI link name. This issue has now been resolved. The Quick Links widget will display the correct link name for OpenAPI specifications.

Reviewer: ```suggestion
In the Cockpit and Device Management application, the "Quick links" widget, which provides easy access to useful links, did not display the link name for the OpenAPI specification properly. This issue has been resolved. The "Quick links" widget now displays the correct link name "OpenAPI specification".
```
